router.js

```diff
@@ -44,10 +44,10 @@ export const installRouter = (locationUpdatedCallback) => {
     e.preventDefault();
     if (href !== location.href) {
       window.history.pushState({}, '', href);
-      locationUpdatedCallback(location);
+      locationUpdatedCallback(location, e);
     }
   });
 
-  window.addEventListener('popstate', () => locationUpdatedCallback(window.location));
-  locationUpdatedCallback(window.location);
+  window.addEventListener('popstate', e => locationUpdatedCallback(window.location, e));
```

Comment:
this change isn't needed, if you just pass `null` below, right? Like the old code sample should still work?

Reply:
Yes, this is a non-breaking change; you can optionally update your callback to use the second argument.